### PR TITLE
Update references to 'activity' to reflect new project

### PR DESCRIPTION
**Pull request by mariel-delagarza**
Update references to "Activity" to be "timelineEvent" -- `activity` was a holdover from the `database-table-template`.

Use `timelineEvent` to reflect original project idea and distinguish between browser events/event handlers.

### CODE REVIEW

File: src/components/Table.svelte [hunk 1]
```diff
@@ -122,10 +122,10 @@
     iconsActive.forEach((icon) => {
       icon.classList.remove("sort-icon--active")
     })
-    const divActivity = document.querySelector(
-      ".table__cell--header__container__activity",
+    const divTimelineEvent = document.querySelector(
+      ".table__cell--header__container__timelineEvent",
```
Comment (jdromero88) [inline src/components/Table.svelte]: I suggest changing this to ```".table__cell--header__container__event"```

File: src/scss/components/_table.scss [hunk 2]
```diff
@@ -93,7 +93,7 @@
         align-items: center;
       }
 
-      &__container__activity,
+      &__container__timelineEvent,
```
Comment (jdromero88) [inline src/scss/components/_table.scss]: I suggest changing this to ```&__container__event```

**Review by jdromero88 — DISMISSED** [review]
Thank you for working on this @mariel-delagarza! I found out this is breaking the sort functionality.
image

On the Table.svelte component In line 159
```html 
class="table__cell--header__container table__cell--header__container__{name
.toLowerCase()
.split(' ')
.join('-')}
```
Here we are looping through the header names, so the div it's being rendered with a class `table__cell--header__container__event`

Also for consistency, let's try to keep the class names with kebab-case please.
I left you a few comments below.

**Comment by mariel-delagarza** [comment]
Gracias! I made these class name changes (to `container__event`) and that fixed the sort functionality 👍

**Pull request merged**